content/articles/ssl-certificates-email-validation.markdown

```diff
@@ -60,8 +60,9 @@ The approval email typically can be sent to the following addresses, called admi
 
 Where `example.com` is the domain for the certificate being purchased.
 
-Alternatively, **the approval email can be sent to a different email address only if this is listed in the WHOIS information for the domain**. In fact, this is the only way for the Certificate Authority to determine if an email is officially associated with a domain.
+## Email validation and GDPR
 
+Due to the new privacy rules enacted by GDPR, you can no longer use a custom email address listed in your WHOIS data as it is masked out for privacy. You will have to select from the provided list above to send the validation email.
```

Comment:
@martinisoft we should perhaps at least add a link to https://en.wikipedia.org/wiki/General_Data_Protection_Regulation (and/or expand what GDPR is).

Reply:
I went with the GDPR alliance explanation which puts it pretty nicely: https://medium.com/@GDPR_alliance/the-general-data-protection-regulation-gdpr-in-a-nutshell-2f02290dbd0d